packages/create-vite/template-react-ts/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc -b && vite build",
+    "build": "tsc && tsc -p tsconfig.node.json && vite build",
```

Comment:
I believe we needed `references` so that the IDE picks up the right config when typechecking the Vite config? Perhaps we need both this script change and the `references` to properly support the usecase, and should be good enough?